.github/workflows/release.yml

```diff
@@ -22,10 +22,12 @@ jobs:
           RELEASE_SSH_USER: ${{ secrets.RELEASE_SSH_USER }}
           RELEASE_SSH_KEY: ${{ secrets.RELEASE_SSH_KEY }}
           SLACK_API_TOKEN: ${{secrets.SLACK_API_TOKEN }}
-        uses: SonarSource/gh-action_release/main@v3backup
+        uses: SonarSource/gh-action_release/main@master
```

Comment:
Should we always use the latest master? It seems to me that all other projects we have do use `main@v3`.